### DISCUSSION TITLE
Do you have Discord community?

### DISCUSSION THREAD

**Discussion by yuki0418** [loco-rs/loco#113] [General]
I'm interested in if you already have a discode community. If so, many people probably want join? (Or maybe still early stage to make a community now)

**Comment by kaplanelad** [comment]
Hello @yuki0418,

While we do have plans to establish a Discord channel in the future. Currently, we're actively engaged in addressing discussions and issues right here.

Please share what features or aspects you find lacking here compared to Discord. 
Your feedback is valuable to us :) 

Thanks

**Reply by yuki0418** [reply]
Hi @kaplanelad 

Thank you for replying!!

>Please share what features or aspects you find lacking here compared to Discord.
In this phase, agree with discussions and issues are good place.

In Stage 4, maybe Discode ore other platforms may be better to spread about Loco framework to other developer to involve into contribution, I guess.

**Reply by kaplanelad** [reply]
Hey @yuki0418,
We just opened a Discord: https://discord.gg/Npcwuvq6

you are more than welcome to join

**Reply by dominiwe** [reply]
@kaplanelad This invite link is expired and not permanent. Additionally, this discord community is not linked anywhere in this github project or the website/docs, right? If the community is active, I'd like to join.

**Reply by kaplanelad** [reply]
we have the kink in readme.
![image](https://github.com/user-attachments/assets/891a272d-0324-47d5-bec3-6c59046a875c)

**Comment by tjyang** [comment]
>While we do have plans to establish a Discord channel in the future. Currently, we're actively engaged in addressing discussions and issues right here.
I think (IMHO) github discussion and issues are more than enough for project communication.
I did join the discord reluctantly but I do feel discord is an extra effort for dev resources to cover.
Question bother me often is  where should I post my comment/issue/question ?

**Reply by kaplanelad** [reply]
@tjyang, This is your choice of where you prefer to talk. If you have any questions, feel free to express them in the discussion or Discord. Should you come across any bugs, please open an issue.

Additionally, Discord provides us with the option to have private conversations if necessary.. Keep it straightforward for yourself—simply choose the platform where you prefer to discuss matters and we will be there
